### PR TITLE
[WIP] Improve specs for charges and rates

### PR DESCRIPTION
**Pull request by mresti**
This PR contains new specs for model showback_charge and showback_rate. I had completed 2 pending specs for model showback_charge.

### CODE REVIEW

File: spec/models/showback_charge_spec.rb [hunk 1]
```diff
@@ -37,6 +37,31 @@
...
+    charge.valid?
+    expect(charge.errors.details[:variable_cost]).to include(:error => :not_a_number, :value => "cost")
+  end
+
+  it "fixed_cost expected to be Money" do
+    expect(FactoryGirl.create(:showback_charge, :fixed_cost => Money.new("5.9234892348923"))).to be_valid
+    expect(ShowbackCharge).to monetize(:fixed_cost)
+  end
+
+  it "variable_cost expected to be BigDeciaml" do
```
Comment (sergio-ocon) [inline spec/models/showback_charge_spec.rb]: Please change the name, this should be now Money, not big decimal

File: spec/models/showback_rate_spec.rb [hunk 2]
```diff
@@ -42,6 +42,30 @@
       expect(showback_rate.errors.details[:dimension]).to include(:error=>:blank)
     end
 
+    it 'is valid when assigning fixed_rate' do
+      showback_rate.fixed_rate = 15
+      showback_rate.valid?
```
Comment (sergio-ocon) [inline spec/models/showback_rate_spec.rb]: Can you please verify that the result after saving it is the same you get?
I got a problem where I was storing 5, but receiving 50, when I was changing from BigDecimal to Money

Comment (sergio-ocon) [inline spec/models/showback_rate_spec.rb]: That should be another test:
showback_... = 15
showback_rate.save.reload
expect... to be (Money(15)

Comment (mresti) [inline spec/models/showback_rate_spec.rb]: Yes, I will verify the result after saving.

File: spec/models/showback_charge_spec.rb (hunk 1 above)
Comment (mresti) [inline spec/models/showback_charge_spec.rb]: Thanks, I forgot it.

**Comment by mresti** [comment]
I want to add the changes in this PR to add your comments.

**Comment by sergio-ocon** [comment]
Please rebase and republish

**Comment by aljesusg** [comment]
rebase this

**Comment by sergio-ocon** [comment]
It seems that it got mixed, we have commits that are not coming from your code...

**Comment by mresti** [comment]
Yes, I did a rebase about the branch consumption and it seem that I did anything wrong :( I would try to fix that.

**Pull request closed without merging**